src/actions/db-connector.ts

```diff
@@ -0,0 +1,22 @@
+import {logger} from '../logger/logger';
```

Comment:
create DAL folder and put there all that related to DB

Reply:
added DAL folder under actions folder.
Should it be outside actions? if so actions folder is unnecessary